patches/tModLoader/Terraria/ModLoader/Core/ModOrganizer.cs

```diff
@@ -138,7 +138,13 @@ private enum SearchFolders { }
 				if (!TryReadManifest(GetParentDir(mod.modFile.path), out var manifest))
 					continue;
 
-				WorkshopHelper.QueryHelper.GetDependenciesRecursive(manifest.workshopEntryId, ref dependencies);
+				try {
+					WorkshopHelper.QueryHelper.GetDependenciesRecursive(manifest.workshopEntryId, ref dependencies);
+				}
+				catch (Exception e) {
+					Utils.ShowFancyErrorMessage($"Failed to fetch missing dependencies for local mod: {mod.DisplayName}. Check your internet connection or download manually from Steam Workshop!", Interface.loadModsID);
+					Logging.tML.Warn($"Failed to fetch missing dependencies for local mod: {mod.DisplayName}. Check your internet connection!", e);
```

Comment:
Should probably be moved to a localization key, and the string value should be the same for both here.

Reply:
What key should I use? `tModLoader.WorkshopFetchDependenciesFailure` ?

Reply:
Sure.